README.md

```diff
@@ -0,0 +1,168 @@
+# Hello-World-Microverse-Week1
+
+<a name="readme-top"></a>
+
+
+
+
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello World Project] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences. This is my first project in Microverse
```

Comment:
- [ ] As the previous reviewer suggested, Kindly ensure to provide a detailed description of the project. This will enable the visitors have a clear understanding of what the project is all about.
- [ ] [OPTIONAL] Kindly delete the guideline text `Describe your project in 1 or 2 sentences`

Reply:
Dear @AlphaNtihinduka, I did the suggestions but for the required changes: "Kindly consider the inline comments", is it only optional suggestion or which inline comments do you mean? Is my project approved now or what changes need to be done yet? because I have only one review request left. That is why I wanted to clarify. Thanks

Reply:
@AFakhriddinov, the previous reviewer meant the suggestions he made which you have done. 👍